src/SUMMARY.md

```diff
@@ -10,6 +10,7 @@
     - [Partitioning Notes](./installation/liveimages/partitions.md)
     - [Installation Guide](./installation/liveimages/guide.md)
   - [Guides](./installation/guides/index.md)
+    - [Installation Guide (chroot)](./installation/guides/chroot-install.md)
```

Comment:
You haven't created the index for the boot loader section.

Reply:
Updated